### PR TITLE
#474  Make rule for stage deploy

### LINKED ISSUE #474: Create stage deploy rule
Now it's too hard to deploy on stage every time. Create make rule to build+deploy stage

### PR DESCRIPTION
**Pull request by duker33**
Closes #474

### CODE REVIEW

File: docker/Makefile [hunk 1]
```diff
@@ -80,6 +80,27 @@ deploy-dev:
 	$(dc) stop app && $(dc) up -d app
 
 
+deploy-stage:
+	# - build section
```
Comment (artemiy312) [inline docker/Makefile]: Let's separate build and deploy receipt

Comment (artemiy312) [reply docker/Makefile]: Please, create todo to build the image in CI on merge into the master branch

File: docker/Makefile [hunk 2]
```diff
@@ -80,6 +80,27 @@ deploy-dev:
...
+	# preserve the dev image from repository
+	$(d) tag fidals/stb:dev fidals/stb:dev_temp
```
Comment (artemiy312) [inline docker/Makefile]: Why do we need to preserve dev image? What do you mean by this?

File: docker/Makefile [hunk 3]
```diff
@@ -80,6 +80,27 @@ deploy-dev:
...
+	$(dcp) stop app-stage
+	$(dcp) rm -f app-stage
```
Comment (artemiy312) [inline docker/Makefile]:
```suggestion
	$(dcp) rm -fs app-stage
```
It will stop and remove the container

File: docker/Makefile [hunk 4]
```diff
@@ -80,6 +80,27 @@ deploy-dev:
...
+	$(MAKE) build-static
+	$(dcb) build --no-cache python-dev python-stage
+	$(d) tag fidals/stb:dev_temp fidals/stb:dev
+	# just untag image, not remove
+	$(d) rmi fidals/stb:dev_temp
+
+	# - deploy section
+	$(dcp) up -d app-stage
+	$(dcp) run --rm app-stage python manage.py migrate
```
Comment (artemiy312) [inline docker/Makefile]:
```suggestion
	$(dcp) exec app-stage python manage.py migrate
```
and below too

File: docker/docker-compose-build.yml [hunk 5]
```diff
@@ -14,6 +14,12 @@ services:
       context: ../
       dockerfile: docker/images/python/Dockerfile.prod
 
+  python-stage:
+    image: fidals/stb:stage
```
Comment (artemiy312) [inline docker/docker-compose-build.yml]: Why do we need separate image for stage? The stage and prod have the same image

File: docker/Makefile (hunk 1 above)
Comment (duker33) [reply docker/Makefile]: @artemiy312 , stage is required to show how random branch works. So, we can't build correct stage on merging to master

File: docker/Makefile (hunk 2 above)
Comment (duker33) [reply docker/Makefile]: @artemiy312 , comment will be too big and complex. I'll update docker-compose-build file instead

File: docker/docker-compose-build.yml (hunk 5 above)
Comment (duker33) [reply docker/docker-compose-build.yml]: @artemiy312 , stage is created to expose given branch different from master.
`fidals/stb:prod` image contains master branch.
`fidals/stb:stage` contains a given one.

I'll provide comment about it

**Pull request merged**